### PR TITLE
Bugfix 831 (Returning Graph Types instead of PSCustomObjects)

### PR DESCRIPTION
**Pull request by v-akarke**
Updated the documentation and tested all the examples; everything is working as expected.

Get-EntraDeviceRegisteredUser
Get-EntraDomainNameReference
Revoke-EntraSignedInUserAllRefreshToken
Get-EntraUserOwnedObject
Get-EntraApplicationOwner
Get-EntraUserManager
Get-EntraGroupOwner
Get-EntraUserDirectReport
Get-EntraDeviceRegisteredOwner
Get-EntraObjectByObjectId
Get-EntraDirectoryRoleMember
Set-EntraTrustedCertificateAuthority
New-EntraTrustedCertificateAuthority
Remove-EntraTrustedCertificateAuthority
Set-EntraBetaTrustedCertificateAuthority
New-EntraBetaTrustedCertificateAuthority
Remove-EntraBetaTrustedCertificateAuthority
New-EntraInvitation

### CODE REVIEW

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraUserDirectReport.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Get-EntraUserDirectReport
+title: Get-EntraUserDirectReport
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraUserDirectReport.md:3]: Remove this

File: module/Entra/customizations/New-EntraTrustedCertificateAuthority.ps1 [hunk 2]
```diff
@@ -7,72 +7,24 @@
...
-            $params["ErrorAction"] = $PSBoundParameters["ErrorAction"]
-        }
-        if($null -ne $PSBoundParameters["WarningAction"])
-        {
-            $params["WarningAction"] = $PSBoundParameters["WarningAction"]
-        }
-        
         $newCert = $PSBoundParameters["CertificateAuthorityInformation"]
-        $previusCerts = @()        
+        $previusCerts = @()
```
Comment (KenitoInc) [inline module/Entra/customizations/New-EntraTrustedCertificateAuthority.ps1:17]:
```suggestion
        $previousCerts = @()
```

Comment (KenitoInc) [reply module/Entra/customizations/New-EntraTrustedCertificateAuthority.ps1:17]: Update all the other variables as appropriate

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraUserDirectReport.md (hunk 1 above)
Comment (v-akarke) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraUserDirectReport.md]: Removed

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**